gateway-service/src/main/java/org/zowe/apiml/gateway/conformance/VerificationOnboardService.java

```diff
@@ -102,54 +89,24 @@ public String getSwagger(String swaggerUrl) {
      * Checks if endpoints can be called and return documented responses
      *
      * @param endpoints                 endpoints to check
-     * @param passedAuthenticationToken Token used to call endpoints that support SSO
      * @return List of problems
      */
-    public List<String> testEndpointsByCalling(Set<Endpoint> endpoints, String passedAuthenticationToken) {
-        ArrayList<String> result = new ArrayList<>(checkEndpointsNoSSO(endpoints));
-        try {
-            result.addAll(checkEndpointsWithSSO(endpoints, passedAuthenticationToken));
-        } catch (ValidationException e) {
-            result.add(e.getMessage());
-        }
-
-        return result;
+    public List<String> testEndpointsByCalling(Set<Endpoint> endpoints) {
+        return checkEndpoints(endpoints);
     }
 
-    private List<String> checkEndpointsWithSSO(Set<Endpoint> endpoints, String passedAuthenticationToken) {
+    private List<String> checkEndpoints(Set<Endpoint> endpoints) {
         ArrayList<String> result = new ArrayList<>();
 
-        String ssoCookie = getAuthenticationCookie(passedAuthenticationToken);
-
-        HttpHeaders headersSSO = new HttpHeaders();
```

Comment:
We cannot remove JWT from the calls. It is necessary to obtain it from the Authentication object (security context). And in the case, JWT is not accessible there (ie. access with x509), obtain a new token.